xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -57,12 +56,12 @@ pub enum InnerApiClient {
     ),
 }
 
-pub struct Client {
+pub struct XmtpGrpcClient {
```

Comment:
I renamed to be a little easier to understand alongside the other `XmtpApiClient`:
`xmtp_api_grpc` -> `XmtpGrpcClient`
`xmtp_api_grpc_gateway` -> `XmtpGrpcGatewayClient`